pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -2652,8 +2652,8 @@ func isMatchAllFilterChain(fc *listener.FilterChain) bool {
 
 func isFallthroughFilterChain(fc *listener.FilterChain) bool {
 	if fc.Metadata != nil && fc.Metadata.FilterMetadata != nil &&
-		fc.Metadata.FilterMetadata[PilotMetaKey].Fields["fallthrough"].GetBoolValue() {
```

Comment:
This could be `		fc.Metadata.FilterMetadata[PilotMetaKey].GetFields()["fallthrough"].GetBoolValue() {` I think since gogo does nil checks